test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/ClientModeRunner.scala

```diff
@@ -32,13 +32,16 @@ object ClientModeRunner {
         s"--master $mesosMasterUrl",
         "--deploy-mode client",
         s"""--driver-java-options "$logFileConfig"""",
-        s"""--conf spark.executor.extraJavaOptions="$logFileConfig""""
-      )
+        s"""--conf spark.executor.extraJavaOptions="$logFileConfig"""")
 
       if (config.hasPath("spark.driver.host")) {
         sparkSubmitJobDesc += s"--conf spark.driver.host=${config.getString("spark.driver.host")}"
       }
 
+      if (config.hasPath("spark.executor.uri")) {
+        sparkSubmitJobDesc += s"""--conf spark.executor.uri="${config.getString("spark.executor.uri")}""""
```

Comment:
This is wrong, the internal quotes should not be there (I fixed it in the ClusterModeRunner but missed it here)